### ISSUE TITLE
Releasing v6.10

### ISSUE THREAD

**Issue by kazu-yamamoto** [kazu-yamamoto/Mew#176]
Since important features have been merged and documentation was radically improved, I guess that we should release v6.10.

@tats Could you get #172 done?

**Comment by tats** [comment]
> @tats Could you get #172 done?

No, it's hard for me for now.  Preparing multiple XOAUTH2 accounts
and debugging needed.  Could you ask someone else?

Anyway, the new release is welcome even #172 not yet fixed,

**Comment by kazu-yamamoto** [comment]
CC: @yoshinari-nomura

**Comment by kazu-yamamoto** [comment]
@yoshinari-nomura #172 is the showstopper for v6.10. Would you improve XAOUTH2? Or may we just release v6.10?

**Comment by yoshinari-nomura** [comment]
Sorry, I missed GitHub notification.

`Support mew-config-alist to use XOAUTH2 for each servers` would need some amount of work.
I'm willing to do that, but I'm afraid get it done before v6.10.

**Comment by kazu-yamamoto** [comment]
@tats Are we ready for making a new release?

**Comment by tats** [comment]
Yes, though we don't need to hurry.

**Comment by kazu-yamamoto** [comment]
Released Mew v6.10 on https://www.mew.org
I will announce Mew v6.10 to the two MLs later.

**Comment by kazu-yamamoto** [comment]
Closing!
